python/dgl/distributed/optim/pytorch/sparse_optim.py

```diff
@@ -35,16 +46,186 @@ def __init__(self, params, lr):
             self._rank = 0
             self._world_size = 1
 
+    def local_state_dict(self):
+        """Return the state pertaining to current rank of the optimizer.
+
+        Returns
+        -------
+        dict
+            Local state dict
+
+        See Also
+        --------
+        load_local_state_dict
+        """
+        local_state_dict = {}
+        local_state_dict["emb_states"] = {}
```

Comment:
as I mentioned before, could we define global variables for these keys: `emb_states`, `world_size`, `ids` and so on? In case we need to change them and could avoid typo.

Reply:
fixed